test/verify/check-journal

```diff
@@ -471,40 +476,49 @@ s.send(b"PRIORITY=3\\nFOO=bar\\n")'
         sel = "#journal-box .cockpit-logline .cockpit-log-message:contains('crashed in __nanosleep()')"
         b.click(sel)
 
-        # Wait until loaded (when delete button is loaded, all is loaded)
-        sel = "#journal-entry-fields .nav .btn-danger"
-        b.wait_visible(sel)
-
-        sel = "#journal-entry-fields .nav .btn-primary:contains('Report')"
+        sel = "table.reporting-table tr:first-child .btn:contains('Report')"
         b.click(sel)
 
-        # jQuery magic on this page updates the whole frame for changing to "Reported" state
-        b.expect_load_frame("cockpit1:localhost/system/logs")
-        sel = "#journal-entry-fields .nav .problem-btn:contains('Reported')"
-        b.wait_visible(sel)
-        self.assertIn("/reports/bthash/123deadbeef", b.attr(sel, 'href'))
+        sel = "table.reporting-table tr:first-child a[href$='/reports/bthash/123deadbeef'"
+        b.wait_present(sel)
 
-        # "Unreport" the problem to test reporting unknown problem
-        m.execute('find /var/spool/abrt -name "reported_to" | xargs rm')
+        # FIXME: Remove this once reportd-0.7.1 lands there.
```

Comment:
This should happen [in the next days](https://bodhi.fedoraproject.org/updates/FEDORA-2019-1c185dd085); after that it will require a fedora-31 image rebuild. If you want to land it earlier, please add an `else:` that checks what happens with earlier reportd versions.

Reply:
“Next days” is “when F31 goes GA”, because it’s frozen (and the preferred target is a no-go). There are different bugs in previous versions, so I’d rather not check for buggy behavior.

Reply:
Ack. Since 0.7.0 seems to make it into the final release, it should at least do something sensible then, or (better yet) have a versioned dependency to >= 0.7.1?

Reply:
The versioned dependency sounds like a good solution. 0.6.x -> 0.7.0 was me adding the ability to cancel tasks, but then Matej noticed that the reporting never actually finishes in the usual case (final state change is never announced, but it technically does the job), so I fixed it up in .1.

Reply:
So I should maybe also test cancellations.

Reply:
reportd 0.7.1 is in Fedora 31 now, so this should be updated now. But I'll put that onto the next review round, to make it a bit easier to track everything.